internal/provider/provider.go

```diff
@@ -90,6 +90,7 @@ func (p *BloxOneProvider) Resources(ctx context.Context) []func() resource.Resou
 func (p *BloxOneProvider) DataSources(ctx context.Context) []func() datasource.DataSource {
 	return []func() datasource.DataSource{
 		ipam.NewIpSpaceDataSource,
+		ipam.NewIpamHostDataSource,
```

Comment:
I don't see an entry in `Resources` function for ipam host

Reply:
Added